dart/lib/src/sentry_measurement_unit.dart

```diff
@@ -23,13 +31,66 @@ enum SentryMeasurementUnit {
   /// Week (`"week"`), 604,800 seconds.
   week,
 
+  /// Information units
+
+  /// Bit (`"bit"`), corresponding to 1/8 of a byte.
+  bit,
```

Comment:
`m`: Can you do nested enums somehow similar to Java? If yes, we could duplicate the existing ones into an inner class or interface enum, mark the old ones as deprecated, and do information and fraction units properly.

```dart
// Pseudo code 
enum SentryMeasurementUnit {

@deprecated
nanoSecond,

...

  enum Duration {
    nanoSecond, // ...
  }

  enum Information {
   // ...
   }

}
```

Reply:
Nope, the description states the limitation of the enum type.

Reply:
See https://dart.dev/guides/language/language-tour#enumerated-types
Before and after `Declaring enhanced enums` section.

Reply:
What about splitting it up into `SentryMeasurementUnit`, `SentryMeasurementInformationUnit`, etc. and marking `SentryMeasurementUnit` as deprecated?